.circleci/config.yml

```
@@ -35,11 +35,14 @@ jobs:
           at: ~/build
       - run:
           name: pack
-          command: dotnet pack -p:PackageVersion=${CIRCLE_TAG} -p:TargetFrameworks=netcoreapp2.2 --configuration Release -o ../../dist
+          command: dotnet pack -p:PackageVersion=${CIRCLE_TAG} -p:TargetFrameworks=netcoreapp2.2 --configuration Release -o ../../dist   
       - persist_to_workspace:
           root: .
           paths:
             - ./dist
+      - store_artifacts:
+          path: ../../dist/Dapper.Azure.WebJobs.Extensions.SqlServer.${CIRCLE_TAG}.nupkg
```

Does this work? I think that the path should be from the "root" of the working directory and not from the src/dapper.... folder..

So I think it should look more like the following? 
```suggestion
          path: ./dist/Dapper.Azure.WebJobs.Extensions.SqlServer.${CIRCLE_TAG}.nupkg
```

---

both works